api/src/modules/content-pages/queries/avo/getNlParentContentPages.graphql

```diff
@@ -0,0 +1,11 @@
+query getNlParentContentPages($limit: Int!) {
```

Comment:
NL will always be the parent page
We need a label or id to group all translated pages into one group.
eg:
* homepage NL
* homepage EN
* homepage FR

So we chose the dutch page id as that grouping id. So that function will never change even if we add more languages

Reply:
Not sure I like that from a design perspective but if it was agreed on; ✅